deploy/iso/minikube-iso/package/podman/podman.mk

```diff
@@ -29,6 +29,8 @@ endef
 
 define PODMAN_INSTALL_TARGET_CMDS
 	$(INSTALL) -Dm755 $(@D)/bin/podman $(TARGET_DIR)/usr/bin/podman
+	$(INSTALL) -d -m 755 $(TARGET_DIR)/etc/cni/net.d/
+	$(INSTALL) -m 644 cni/87-podman-bridge.conflist $(TARGET_DIR)/etc/cni/net.d/87-podman-bridge.conflist
```

Comment:
is this something we could add to ubuntu image too ? kicbase?

Reply:
As far as I know, it is already included (provided by `podman` package):

```console
docker@minikube:~$ dpkg -S /etc/cni/net.d/87-podman-bridge.conflist 
podman: /etc/cni/net.d/87-podman-bridge.conflist
```

If you do a `sudo make install` in libpod, it will be included by default.

Reply:
Seems to work OK there, with the provided CNI:

```console
docker@minikube:~$ sudo podman run -it busybox
Trying to pull docker.io/library/busybox...
Getting image source signatures
Copying blob e2334dd9fee4 done  
Copying config be5888e67b done  
Writing manifest to image destination
Storing signatures
/ # 
```

So this was only a missing file on the minikube VM